travis-build/test.sh

```diff
@@ -14,6 +14,11 @@
 
 set -o pipefail
 
+# start etcd
+etcd &
```

Comment:
I think kill it seems better. So I don't need to kill it myself when I run the script on my machine.